src/main/kotlin/solve/importer/ProjectParser.kt

```diff
@@ -20,8 +20,6 @@ object ProjectParser {
     private const val JPG_EXTENSION = "jpg"
     private val possibleExtensions = setOf(PNG_EXTENSION, JPG_EXTENSION)
 
-    private val importer = find<ImporterView>()
```

Comment:
Why it can not be found one time?

Reply:
this causes problems with displaying alerts